game/dota_addons/dota_imba/scripts/vscripts/hero/hero_phantom_lancer.lua

```diff
@@ -0,0 +1,206 @@
+
+	-- Ability specials
+	local illusion_lance_cast_range = ability:GetSpecialValueFor("illusion_lance_cast_range")
+
+	-- Find all nearby illusions
+	local units = FindUnitsInRadius(caster:GetTeamNumber(),
+	 									caster:GetAbsOrigin(),
+	 									nil,
+	 									illusion_lance_cast_range,
+	 									DOTA_UNIT_TARGET_TEAM_FRIENDLY,
+	 									DOTA_UNIT_TARGET_HERO,
+	 									DOTA_UNIT_TARGET_FLAG_PLAYER_CONTROLLED,
+	 									FIND_ANY_ORDER,
+	 									false)	
+
+	-- Cycle between found units, only include illusions
+	for _, unit in pairs(units) do
+		if unit:GetPlayerOwnerID() == caster:GetPlayerOwnerID() and unit ~= caster then			
+
+			-- Fake casting animation
+			unit:StartGesture(ACT_DOTA_CAST_ABILITY_1)				
+
+			-- Illusions that are moving/attacking are ignored. Idle illusions fake their Spirit Lance
+			if unit:IsIdle() then
+				unit:FaceTowards(target:GetAbsOrigin())
+			end			
+		end
+	end
+
+	return true
+end
+
+function imba_phantom_lancer_spirit_lance:OnSpellStart()
+	-- Ability properties
+	local caster = self:GetCaster()
+	local ability = self
+	local target = self:GetCursorTarget()
+	local sound_cast = "Hero_PhantomLancer.SpiritLance.Throw"
+	local projectile_lance = "particles/units/heroes/hero_phantom_lancer/phantomlancer_spiritlance_projectile.vpcf"
+
+	-- Ability specials
+	local projectile_speed = ability:GetSpecialValueFor("projectile_speed")
+	local illusion_lance_cast_range = ability:GetSpecialValueFor("illusion_lance_cast_range")	
+	local scepter_bounce_count = ability:GetSpecialValueFor("scepter_bounce_count")
+
+	-- Play cast sound
+	EmitSoundOn(sound_cast, caster)
+
+	-- Fire projectile at target	    
+    local lance_projectile
+    lance_projectile =	 {Target = target,
+                          Source = caster,
+                          Ability = ability,
+                          EffectName = projectile_lance,
+                          iMoveSpeed = projectile_speed,
+                          bDodgeable = true, 
+                          bVisibleToEnemies = true,
+                          bReplaceExisting = false,
+                          bProvidesVision = false,
+                          ExtraData = {real_lance = 1, bounces_left = scepter_bounce_count}
+    }
+
+    ProjectileManager:CreateTrackingProjectile(lance_projectile)
+
+	-- Nearby illusions also cast their own projectiles. Those are fake, of course
+	-- Find all nearby illusions
+	local units = FindUnitsInRadius(caster:GetTeamNumber(),
+	 									caster:GetAbsOrigin(),
+	 									nil,
+	 									illusion_lance_cast_range,
+	 									DOTA_UNIT_TARGET_TEAM_FRIENDLY,
+	 									DOTA_UNIT_TARGET_HERO,
+	 									DOTA_UNIT_TARGET_FLAG_PLAYER_CONTROLLED,
+	 									FIND_ANY_ORDER,
+	 									false)	
+
+	-- Cycle between found units, only include illusions
+	for _, unit in pairs(units) do
+		if unit:GetPlayerOwnerID() == caster:GetPlayerOwnerID() and unit ~= caster then			
+			lance_projectile =	 {Target = target,
```

Comment:
Looks like there's extra space between the `=` and the `{`, you could remove the spacing and add a newline before `Target =` so it's on its own line above `Source =`

Reply:
Sure, I'll do that.